operations/experimental/README.md

```diff
@@ -9,12 +9,113 @@ This is the README for Experimental Ops-files. To learn more about `cf-deploymen
 - For Community Ops-files, check out the [Community Ops-file README](../community/README.md).
 - For Legacy Ops-files, checkout the [Legacy Ops-file README](../legacy/README.md).
 
-"Experimental" ops-files represent configurations
-that we expect to promote to blessed configuration eventually,
-meaning that,
-once the configurations have been sufficiently validated,
-they will become part of cf-deployment.yml
-and the ops-files will be removed.
+
+# Promoting/Deprecating experimental ops files
+"Experimental" ops-files represent configurations that we expect to promote to blessed configuration eventually, meaning that, once the configurations have been sufficiently validated, they will become part of cf-deployment.yml and the ops-files will be removed. The following workflow illustrates the sequence of events to promote an experimental ops file.
```

Comment:
"once the configurations have been sufficiently validated, they will become part of cf-deployment.yml"

consider changing to:

"once the configurations have been sufficiently validated, they will either become default (inlined into the base manifest, cf-deployment.yml), or GA'd as an optional feature (promoted from experimental to operations directory)."